dockerfile/torch1.14-cuda11.8.dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nvcr.io/nvidia/pytorch:22.12-py3
+
+# Ubuntu: 20.04
```

Comment:
`# Ubuntu: 22.12`

Reply:
the Ubuntu OS for the image is 20.04.5 LTS, 22.12 is just the version/release date for the nvcr image

Reply:
I see. Thanks!